reframe/schemas/junit.xsd

```diff
@@ -5,7 +5,7 @@
 	 attributeFormDefault="unqualified">
 	<xs:annotation>
 		<xs:documentation xml:lang="en">JUnit test result schema for the Apache Ant JUnit and JUnitReport tasks
-Copyright © 2011, Windy Road Technology Pty. Limited
+Copyright 2011, Windy Road Technology Pty. Limited
```

Comment:
I don't like very much that we need to change this. What type of error where you getting?

Reply:
I got UnicodeDecodeError, let me retry.

Reply:
can not reproduce it, reverting to original: 42285eb

Reply:
Yes, because your locale could have been the ANSI C. But to avoid such failures, you can force the encoding in `open()` by passing `encoding='utf-8'`.